docs/powered-by-mesos.md

```diff
@@ -33,6 +33,7 @@ layout: documentation
 * [eBay](http://www.ebay.com)
 * [Ericsson](http://www.ericsson.com/)
 * [Foursquare](http://www.foursquare.com)
+* [GenOuest core facility](http://www.genouest.org)
```

Comment:
Can we omit "core facility" here? It seems inconsistent with the rest of the items in this list, where we've just included the organization name.